pyaptamer/aptanet/_feature_classifier.py

```diff
@@ -144,12 +150,29 @@ def fit(self, X, y):
         self.pipeline_.fit(X, y)
         return self
 
-    def predict(self, X):
+    def predict(self, X, output_type: str = "class"):
+        """
+        Parameters
+        ----------
+        output_type : str, default="class"
+            Type of output to return. Either "class" for class labels or "proba" for
+            (raw) probabilities.
+        """
```

Comment:
Why not mention what one can get for outputs, specially given there are 2 possible outputs